pointgrey_camera_description/urdf/pointgrey_flea3.urdf.xacro

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="pointgrey_flea3">
+  <!-- camera_{x,y,z,mass} are they physical properties of the camera since
+       flea3's can have different physical footprints.  -->
+  <xacro:macro name="pointgrey_flea3"
+    params="frame:=camera name:=camera camera_x camera_y camera_z camera_mass
+             hfov fps width height format:=R8G8B8 near:=0.5 far:=300">
```

Comment:
There are many different options for the [Flea3](http://www.ptgrey.com/flea3-ieee-1394b-firewire-cameras).  Adding a lens will change the physical size and optical properties.